src/rnp/rnp.cpp

```diff
@@ -856,8 +866,8 @@ main(int argc, char **argv)
                 rnp_cfg_setstr(&cfg, CFG_SSHKEYFILE, optarg);
                 break;
             case 'V':
-                print_praise();
-                exit(EXIT_SUCCESS);
+                setcmd(&cfg, CMD_VERSION, optarg);
```

Comment:
Shouldn't this check the return value now?

Reply:
Generally - yes. I allowed it here because CMD_VERSION/CMD_HELP always succeed. But you are right, it is better to add checks.

Reply:
Argh, sorry, misunderstood the question. Here it will just set the command, return value will be checked below (the next your comment).

Reply:
Omg, no. I will get back to this tomorrow. Too late at the evening to think well.
Only now got what did your comment actually mean :-)